src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -65,7 +65,7 @@
 
   <!-- Publish projects if they were not published after build -->
```

Comment:
Did you also want to update this comment (similar to the one in PublishedProjectPackage.targets), or is that not appropriate here?

Reply:
Done